setup.py

```diff
@@ -25,7 +23,7 @@
     "ctapipe-stage1 = ctapipe.tools.stage1:main",
     "ctapipe-merge = ctapipe.tools.dl1_merge:main",
 ]
-tests_require = ["pytest"]
+tests_require = ["pytest", "pandas>=0.24.0"]
```

Comment:
isn't pandas still used in some of the notebooks generated by the docs?

Reply:
ha, yes, probably, will also add to `docs`

Reply:
that's another issue of course, we could probably clean them up to only use astropy

Reply:
seems not to be the case, the docs build passes...

Reply:
or the `read_table` function, yes.

Reply:
the docs build installed `[all]` not just `[docs]`.